### ISSUE TITLE
RECOV on own data

### ISSUE THREAD

**Issue by Rosadosa** [datduong/RECOV#1]
Hello Dat Duong,

with great interest I'm reading your paper on RECOV, after seeing your talk at ASHG.
I'm trying to apply it on my own data after running PrediXcan on all the GTEx tissues. 
This outputs already a gene p-value for each gene in each tissue, however these are not independent and I would like to get the multi-tissue p-value. Do you think your method would be suitable for this?
Can I use gene p-value inputs instead of SNP values? Or would a normal meta make more sense in this case?
Would love to hear your thoughts.

Best,
Roos

**Comment by datduong** [comment]
Meta-analysis method relies on the observed test statistics (like regression coefficients) to be normally distributed. A observed test statistics like a p-value is not normally distributed, and also, you won't have the variance for such observed test statistics. The underlying theory won't work right if you use p-values. Is it possible to convert the p-values back to regression coefficients?

**Comment by Rosadosa** [comment]
Sorry i was not clear, I do have the p-value for the gene with the coefficients.
I have these columns in my file: 
gene | beta | t | p | se(beta)

**Comment by datduong** [comment]
Do you mean that you have the beta coefficient for every cis-snp (or some other set of snps) with respect to the same gene? In this current setup here, the code only takes in a list of beta coefficients for every cis-snp w.r.t. the same gene.

**Comment by Rosadosa** [comment]
No, I have the gene p-value, so I guess I cannot use the program, because I saw that it needs the LD between the SNPs as input as well.
Thanks for your time anyway!

**Comment by datduong** [comment]
Sure, no problem, feel free to reach out if you have any questions related to this types of meta-analysis problem.

**Comment by Rosadosa** [comment]
Well the problem I have is related to your paper in the following way that the tissues in GTEx are of course related, not only multiple tissues per person (and thus the same genetic background), but also that tissues have similar functions and thus most likely similar expression patterns.  

When I have the gene p-value output of all the tissues individually, to perform a meta (to see if a gene is up- or downregulated in cases or controls in all tissues) would not be correct because of this relatedness. However, I am struggling to find a way to combine the p-values, to get an overall expression difference, that's when I found your paper and thought maybe it would be useful.
However, I do not have sufficient background to understand if your method would also work on genes if it would be tweaked a bit.

**Comment by datduong** [comment]
Let me discuss the problem with my advisor and labmates to see how the code can be tweaked.
